pkg/util/helm/helm.go

```diff
@@ -127,6 +140,21 @@ func KlusterToHelmValues(kluster *v1.Kluster, secret *v1.Secret, kubernetesVersi
 		hashedPassword = string(hashedBytes)
 	}
 
+	dex := dexValues{
```

Comment:
actually, since connectors have `omitempty` in annotation, it is not serialized into the values when both are `false` and thus it did not break the default case.  

but your suggestion is more explicit, I have fixed it, thanks! :)

Reply:
The thing is a boolean is never empty. its either `false` or `true` so you can't really "omit" a boolean struct field during serialisation.